internal/cmd/refactortests/cmd/refactortests.go

```diff
@@ -0,0 +1,364 @@
+				fmt.Fprintln(os.Stderr, errors.Wrap(err, "encoding error while running test script"))
+				cause := errors.Cause(err)
+				if e, ok := cause.(*testing.AssertEqualsError); ok {
+					aee = e
+				}
+			}
+		}
+	}
+
+	diff := diffBuf.String()
+	if aee != nil && len(diff) == 0 {
+		// populate diff because there was an assertion error
+		// the caller should know there was a difference, indeed.
+		diff = aee.Error()
+	}
+
+	return resultBuf.String(), diff, nil
+}
+
+// inPlaceTestGen adds the statements to run the test cases in pkg.
+// It doesn't have side effects on pkg, rather, it returns a new, edited one.
+// It avoids to run the code generation step.
+func inPlaceTestGen(pkg *ast.Package) (*ast.Package, error) {
+	testPkg := pkg.Copy().(*ast.Package)
+	// make it a main package for execution
+	testPkg.Package = "main"
+	// find and edit TestStatements
+	pattern := &ast.File{
+		Body: []ast.Statement{
+			&ast.TestStatement{},
+		},
+	}
+	files := edit.Match(testPkg, pattern, true)
+
+	for _, file := range files {
+		f := file.(*ast.File)
+		for _, stmt := range f.Body {
+			if ts, ok := stmt.(*ast.TestStatement); ok {
+				fnCall := &ast.ExpressionStatement{
+					Expression: &ast.CallExpression{
+						Callee: &ast.MemberExpression{
+							Object:   &ast.Identifier{Name: "testing"},
+							Property: &ast.Identifier{Name: "run"},
+						},
+						Arguments: []ast.Expression{
+							&ast.ObjectExpression{
+								Properties: []*ast.Property{
+									{
+										Key:   &ast.Identifier{Name: "case"},
+										Value: ts.Assignment.ID,
+									},
+								},
+							},
+						},
+					},
+				}
+				f.Body = append(f.Body, fnCall)
+			}
+		}
+	}
+
+	return testPkg, nil
+}
+
+func updateDataOrReload() (int, error) {
+	fmt.Println("Do you want to edit the expected result (1) or change the file and reload it from disk (2)? (any other to exit)")
+	scanner := bufio.NewScanner(os.Stdin)
+	scanner.Scan()
+	if scanner.Err() != nil {
+		return -1, scanner.Err()
+	}
+	choice, _ := strconv.Atoi(scanner.Text())
+	return choice, nil
+}
+
+func updateOutData(node *ast.Package, rawResult string) error {
+	as, err := getOutDataVariable(node)
+	if err != nil {
+		return err
+	} else if as == nil {
+		// no assignment chosen, stop
+		return nil
+	}
+
+	as.Init = &ast.StringLiteral{Value: "\n" + rawResult}
+	fmt.Println("Data has been updated")
+	return nil
+}
+
+func updateScript(fname string, script *ast.Package) error {
+	fmt.Printf("Content of the new script:\n\n%s\n\n", ast.Format(script))
+	y, err := yesOrNo("Do you want to update the script (y/n)?")
+	if err != nil {
+		return err
+	}
+	if !y {
+		fmt.Println("Script NOT updated")
+		return nil
+	}
+	err = ioutil.WriteFile(fname, []byte(ast.Format(script)), 0644)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s updated\n", fname)
+	return nil
+}
+
+func getOutDataVariable(node ast.Node) (*ast.VariableAssignment, error) {
+	fmt.Println("Name for the variable holding the expected result:")
```

Comment:
(minor comment) You could offer the user a default of `outData` so they can just press enter here if they want.